jetson/percep/main.cpp

```diff
@@ -143,23 +142,25 @@ int main() {
 
         //Run Obstacle Detection
         pointcloud.pcl_obstacle_detection();  
-        obstacle_return obstacle_detection (pointcloud.bearing, pointcloud.distance);
+        obstacle_return obstacleOutput (pointcloud.leftBearing, pointcloud.rightBearing, pointcloud.distance);
 
         //Outlier Detection Processing
         outliers.pop_back(); //Remove outdated outlier value
 
-        if(pointcloud.bearing > 0.05 || pointcloud.bearing < -0.05)
+        if(pointcloud.leftBearing > 0.05 || pointcloud.leftBearing < -0.05)
             outliers.push_front(true);//if an obstacle is detected in front
         else 
             outliers.push_front(false); //obstacle is not detected
 
         if(outliers == checkTrue) //If past iterations see obstacles
-            lastObstacle = obstacle_detection;
+            lastObstacle = obstacleOutput;
         else if (outliers == checkFalse) // If our iterations see no obstacles after seeing obstacles
-            lastObstacle = obstacle_detection;
+            lastObstacle = obstacleOutput;
 
-        obstacleMessage.distance = lastObstacle.distance; //update LCM distance field
-        obstacleMessage.bearing = lastObstacle.bearing; //update LCM bearing field
+        //Update LCM 
+        obstacleMessage.bearing = lastObstacle.leftBearing; // Update LCM bearing field
+        obstacleMessage.rightBearing = lastObstacle.rightBearing;
+        obstacleMessage.distance = lastObstacle.distance; // Update LCM distance field
         #if PERCEPTION_DEBUG
             cout << "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!Path Sent: " << obstacleMessage.bearing << "\n";
```

Comment:
Is there a better way than this string of !!! lol

Reply:
We're going to clean this up in a different PR